src/components/DialogFailedAddProduct.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
```

Comment:
What all of this have in special with add product? File name can be just "DialogFailed.svelte"